### ISSUE TITLE
How to use in Ron config files.

### ISSUE THREAD

**Issue by patrickisgreene** [AndreaCatania/amethyst_phythyst#12]
Is it possible to add a RigidBodyDesc and ShapeDesc to a component in a prefab ron file?

**Comment by AndreaCatania** [comment]
Yes, unfortunately I don't have any ready example to show you how do it.

**Comment by rosenjcb** [comment]
@AndreaCatania If you don't have any examples, would you mind stating which piece of code dictates how Prefabs are defined in RON files? E.g. is it a simple `derive` statement for `PrefabData` or you do a more manual approach?
